spec/index.bs

```diff
@@ -1081,24 +1081,23 @@ Given an {{HTMLScriptElement}} (|script|), this algorithm performs the following
     If the algorithm threw an error, rethrow the error.
 
 ## Get Trusted Types-compliant attribute value ## {#validate-attribute-mutation}
-To <dfn abstract-op export>get Trusted Types-compliant attribute value</dfn> on {{Attr}} |attribute| with {{Element}} |element| and {{TrustedType}} or a string |newValue|, perform the following steps:
+To <dfn abstract-op export>get Trusted Types-compliant attribute value</dfn> on {{Attr}} |attribute| with {{Element}} |element|, {{TrustedType}} or a string |newValue|, and string |sink|, perform the following steps:
```

Comment:
Instead of the sink being HTMLIFrameElement srcdoc it will be something like `Element setAttribute` or `Attr value` this matches Chrome's behaviour.

Reply:
Need to double check which is correct though does Chrome's impl need updating or is the spec currently just wrong in this case?

Reply:
Though this change does mean the DOM integration needs to wire through the sinks from the IDL all the way down to the call out to this spec?